plugins/CoreHome/lang/en.json

```diff
@@ -82,7 +82,7 @@
         "PivotBySubtable": "This report is not pivoted %1$s Pivot by %2$s",
         "SystemSummaryWidget": "System Summary",
         "SystemSummaryNWebsites": "%d websites",
-        "SystemSummaryNSegments": "%d segments",
+        "SystemSummaryNSegments": "%1$d segments (%2$s pre-processed, %3$s processed in real-time)",
```

Comment:
>  54 segments (7 pre-processed, 47 processed in real-time)

could we maybe rename `%2$s pre-processed` to something like `%2$s processed in background` or so? Although I reckon that doesn't make it much more clear what it means either.

Reply:
What about `processed periodically`? Or `reports generated periodically` vs `reports generated in real-time`?

Reply:
I reckon we can actually just leave it as it is. I don't think any other wording makes it much clear. Same for `processed via cron` etc.